docs/reports/requirements/1_proj-desc.md

```diff
@@ -62,4 +62,38 @@ Note: the User section will be color-coded to show what's already in place and w
 
 ## PDF Report
 
-I will write my PDF report section here to later be incorporated into the full document.
+### High-level Description (about one paragraph)
+
+The objective for the Speaking Portal Project is to design, develop, and deploy a lip-sync animation (LSA) API for the
+Kukarella Table Reads text-to-speech (TTS) web application. This API will serve as an animation generating add-on for
```

Comment:
Maybe I'm misunderstanding, but I believe it's not necessarily meant to be a part of the table reads system, but rather its own kukarella plugin

Reply:
Ah, I think I remember Nazim showing us the Table Reads as an example, but I think you're right that it would be a possible thing just in general. I'll fix this